docs/source/search-syntax.rst

```diff
@@ -148,7 +148,17 @@ search programmatically. You can specify the list of columns to order by
 optional ``DESC`` or ``ASC`` value; the default is ``ASC``. The default ordering is to sort by 
 ``start_time DESC``, then ``run_id``.
 
-For example, to get all `active` runs from experiments IDs 3, 4, and 17 that used a CNN model
+For example, to extract the best `active` run from experiment ID 0 by MAE, use:
```

Comment:
```suggestion
For example, say you want to identify the best model for production deployment. You can use search to find the best `finished` run from experiment ID 0 by accuracy:
```

(was thinking it might actually be good to pass `ViewType.FINISHED` here & search across finished runs, because the likely use case for this is to find a model to deploy etc, so you may only want to include models from finished training runs)